icu4j/main/tests/core/src/com/ibm/icu/dev/test/message2/MessageFormat2Test.java

```diff
@@ -0,0 +1,396 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.dev.test.message2;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import com.ibm.icu.dev.test.TestFmwk;
+import com.ibm.icu.message2.FormattedPlaceholder;
+import com.ibm.icu.message2.Formatter;
+import com.ibm.icu.message2.FormatterFactory;
+import com.ibm.icu.message2.MessageFormatter;
+import com.ibm.icu.message2.Mf2FunctionRegistry;
+import com.ibm.icu.number.FormattedNumber;
+import com.ibm.icu.number.LocalizedNumberFormatter;
+import com.ibm.icu.number.NumberFormatter;
+import com.ibm.icu.util.BuddhistCalendar;
+import com.ibm.icu.util.Calendar;
+import com.ibm.icu.util.GregorianCalendar;
+import com.ibm.icu.util.Measure;
+import com.ibm.icu.util.MeasureUnit;
+
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("javadoc")
+public class MessageFormat2Test extends TestFmwk {
```

Comment:
Please add a brief javadoc comment (`/** ... */`) for each of the `extends TestFmwk` test classes, saying briefly what they do.

Reply:
Done.